releases/handbook.md

```diff
@@ -155,26 +155,25 @@ The manifests repo will be following the release process below:
 
 
 ## Timeline
-
-| Week | Events |
-| --- | --- |
-| 1 | Development |
-| 2 |   |
-| 3 |   |
-| 4 |   |
-| 5 |   |
-| 6 |   |
-| 7 |   |
-| 8 |   |
-| 9 |   |
-| 10 |   |
-| 11 | Feature Freeze, Documentation |
-| 12 |   |
-| 13 | Manifests testing week |
-| 14 | Distributions testing |
-| 15 |   |
-| 16 |   |
-| 17 | Release |
+| **Week** | **Who** | **What** |
+| -------- | ------- | -------- |
+| Week 0 | Release Manager | [Preparation](#preparation) |
+| Week 1 | Release Manager | Start of Release Cycle |
+| Week 1 | Community | [Development](#development-10-weeks) |
+| Week 10 | Release Team | [Feature Freeze](#feature-freeze-2-weeks) |
+| Week 10 | Manifest WG | [rc.0 Released](#feature-freeze-2-weeks) |
+| Week 10 | Docs Lead | [Documentation](#documentation) |
+| Week 10 | Manifest WG | [rc.1 Released](#feature-freeze-2-weeks) |
```

Comment:
According to the [handbook](https://github.com/kubeflow/community/blob/master/releases/handbook.md#feature-freeze-2-weeks), I believe rc.1 is released 2 weeks after the feature freeze? 

I think rc.1 should happen in week 12 since there is a schedule for rc.0 during week 10 already.

Reply:
Nice catch! Yes you are right, it should happen on week 12 which is before the Manifests Testing starts.

I think I'll need to rephrase this sentence in the [Feature Freeze (handbook)](https://github.com/kubeflow/community/blob/master/releases/handbook.md#feature-freeze-2-weeks)
> On the last day of feature freeze, cut a new vX.Y-rc.1 RC tag on the vX.Y-branch release branch.

To make it clear that:
1. `Feature Freeze` does not stop at that time, it continues through the subsequent phases
2. We cut the RC1 because we are preparing for `Manifests Testing`, and not because `Feature Freeze` stops [which doesn't happen]. We want the Manifests leads to use an RC before they start their testing

Reply:
Oh and also I think we'll need to fix the week 10 -> week 12 in this one as well
https://github.com/kubeflow/community/tree/master/releases/release-1.5#timeline

I'll push a commit with this as well

Reply:
That sounds good!

Reply:
I believe https://github.com/kubeflow/community/tree/master/releases/release-1.5#timeline will be fixed with https://github.com/kubeflow/community/pull/538.